src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1357,7 +1357,8 @@ Please specify one of them using the `--source` option to proceed.</value>
     <value>Portable install failed; Cleaning up...</value>

```

Comment:
I really want to sort this file alphabetically but I know VS is just going to mess it up again...